src/components/ScholarshipList.jsx

```diff
@@ -39,6 +40,15 @@ const useStyles = makeStyles((theme) => ({
 
 function ScholarshipList({ scholarships }) {
   const classes = useStyles();
+  const [shareDialogOpen, setShareDialogOpen] = useState(false);
+  const showShareDialog = () => setShareDialogOpen(true);
+  const handleClose = () => setShareDialogOpen(false);
+
+  const [siteLink, setsiteLink] = React.useState('');
```

Comment:
you should add `share` or `shareDialog` in the name to make it clear this is for the share dialog.

Reply:
ok